README.md

```diff
@@ -270,6 +274,38 @@ file. As an example
 
 
 
+### pgenv build-git
+
+This command fetches the current source tree from the official PostgreSQL git repository
+and builds it as special version `dev`. This is an option aimed to help developers
+to keep a version up-to-date with the current PostgreSQL source tree.
+
+Actually, the command is an *alias* for `build dev`, meaning that the following
+two commands are identical:
+    
+    $ pgenv build-git
+    $ pgenv build dev
+    
+The first time the command is called a source directory `dev` is created and the
+default branch is checked out. On a further execution, the command
+will fetch updates from the repository.
+
+It is possible to configure both the repository and the branch to fetch
+using the variables:
+- `PGENV_GIT_REPO` which defaults to `https://git.postgresql.org/git/postgresql.git`;
+- `PGENV_GIT_CHECKOUT_BRANCH` which if not set defaults to fetching the `master` branch, while
```

Comment:
I think it doesn't need the `checkout` bit, or even "git" (what if the project changes VCSes one day?; I suggest calling them `PGENV_PGSQL_REPO` and `PGENV_PGSQL_BRANCH`.

Reply:
Variables renamed, documentation updated.